### ISSUE TITLE
reproducibility of the results

### ISSUE THREAD

**Issue by hedwdwdw** [valeoai/ADVENT#17]
I'm having trouble reproducing the results if I normalize the input image with mean=(0.485, 0.456, 0.406), std=(0.229, 0.224, 0.225) and use the pretrained resnet-101 model on pytorch official website, could you please explain the reason, thanks.
